public/data/1845062-2724/info.json

```diff
@@ -0,0 +1,51 @@
+{
+  "user": {
+    "username": "@userMurllo",
+    "profileImage": "https://campusvirtual.univalle.edu.co/moodle/theme/image.php/mooveuv/core/1729288319/u/f1",
+    "socialLink": "@userMurllo"
+  },
+  "topics": [
+    {
+      "topic": "Vocabulary",
+      "questions": [
+        {
+          "question": "What is the correct translation of 'biblioteca' in English?",
+          "options": ["Library", "Bookstore", "Office", "Museum"],
+          "answer": "Library"
+        },
+        {
+          "question": "Which word is a synonym for 'angry'?",
+          "options": ["Happy", "Furious", "Calm", "Excited"],
+
+          "answer": "Happy"
```

Comment:
@jcv-dev The answer to this question is not accurate enough,  Could you please fix it ? :)